src/dispatch/task/service.py

```diff
@@ -73,6 +75,19 @@ def create(*, db_session, task_in: TaskCreate) -> Task:
 
     assignees = []
     for i in task_in.assignees:
+        participant = participant_service.get_by_incident_id_and_email(
+            db_session=db_session, incident_id=incident.id, email=i.individual.email
+        )
+
+        if not participant or not participant.active_roles:
+            # send emphemeral message to user
```

Comment:
I'd either delete this or add more context to the above line stating that we want to give users context as to why they are being added to an incident.